cmd/action/apply.go

```diff
@@ -53,6 +54,7 @@ func init() {
 	ApplyCmd.Flags().BoolVarP(&ApplyFlags.DryRun, "dry-run", "", false, "Dry-run. Print SQL plan without prompting for execution")
 	ApplyCmd.Flags().StringVarP(&ApplyFlags.Addr, "addr", "", "127.0.0.1:5800", "used with -w, local address to bind the server to")
 	ApplyCmd.Flags().StringSliceVarP(&ApplyFlags.Schema, "schema", "s", nil, "Set schema name")
+	ApplyCmd.Flags().BoolVarP(&ApplyFlags.AutoApprove, "approve", "", false, "Auto approve. Apply the schema changes without prompting for approval")
```

Comment:
I'd try to be more in line what flags like this are usually called. So I propose for `--force` / `-f` or `--no-interaction` or something similar.

Reply:
Tbh I think for the apply command we can stick to the terraform lingo, since users coming from there will feel at home we can do :

![image](https://user-images.githubusercontent.com/1522681/151658346-12edcb3e-fbc5-4b37-8cf6-acc809bdd940.jpeg)

Reply:
I went with `--auto-approve` because I felt like `--force` / `-f` would conflict with the existing `-f` argument. Open to other ideas too